contracts/V1/StateChannelDiamondProxy/DisputeVerificationFacet.sol

```diff
@@ -310,12 +336,42 @@ contract DisputeVerificationFacet is StateChannelCommon {
             UtilityFacet(utilityFacetAddress).concatExitChannelArrays(slashExitChannels, removalExitChannels);
         outputState.totalWithdrawals = _calculateTotalWithdrawals(outputState.totalWithdrawals, allExitChannels);
 
-        outputState.exitBlock =
-            _formExitChannelBlock(latestStateSnapshot.snapshotData.latestExitChannelBlockHash, allExitChannels);
+        Message[] memory outboundMessages = new Message[](allExitChannels.length);
+        for (uint256 i = 0; i < allExitChannels.length; i++) {
+            outboundMessages[i] = Message({
+                messageType: MESSAGE_TYPE_EXIT,
+                participant: allExitChannels[i].participant,
+                balance: allExitChannels[i].balance,
+                data: abi.encode(allExitChannels[i])
+            });
+        }
+        MessageBlock memory outboundMessageBlock;
+        outboundMessageBlock.previousBlockHash = latestStateSnapshot.snapshotData.latestOutboundMessageBlockHash;
+        uint256 outboundHeight = latestStateSnapshot.snapshotData.latestOutboundMessageBlockHeight;
+        if (outboundMessages.length > 0) {
+            outboundMessageBlock.blockHeight = outboundHeight + 1;
+        } else {
+            outboundMessageBlock.blockHeight = outboundHeight;
+        }
+        outboundMessageBlock.messages = outboundMessages;
+        outboundMessageBlock.totalBalance = outputState.totalWithdrawals;
+        outboundMessageBlock.timestamp = 0; // timestamp is not relevant, but more importantly this needs to be deterministic
```

Comment:
There was a bug here - I had to find it manually 
This was `timestamp = block.timestamp` which caused non-determinism during `auditing` since `block.timestamp` changes